astropy/modeling/tests/test_rotations.py

```diff
@@ -16,7 +16,9 @@
 
 
 @pytest.mark.parametrize(('inp'), [(0, 0), (4000, -20.56), (-2001.5, 45.9),
-                                   (0, 90), (0, -90), (np.mgrid[:4, :6])])
+                                   (0, 90), (0, -90), (np.mgrid[:4, :6]),
+                                   [np.random.random((3,4  )) for _ in range(2)],
```

Comment:
If you have to use random numbers, please use a seed so that it is reproducible. I would avoid any randomness altogether if possible.